### ISSUE TITLE
WCAG 2.2 Violation - There are headings present within the interface that are not marked up as headings.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#15]
### Issue

There are some elements that look like headings (for example, that visually convey the start of a section of content) that are not marked up as headings. 

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships)

### Instances

1. #24 

### User Impact

Some users will be unable to access the page's content as expected. For example: the headings will not be announced as headings to screen reader users; users who rely on navigation by heading will be unable to do so; etc..

#### Further information:

1. ["G141: Organizing a page using headings | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Techniques/general/G141)
2. ["Web Accessibility Perspectives: Clear Layout and Design" by W3C Web Accessibility Initiative (WAI) (YouTube) (links to the video in a broader playlist context)](https://www.youtube.com/watch?v=tfkzj5VC9P8&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=6)
3. ["Web Accessibility Perspectives: Clear Layout and Design - Audio Described Version" by W3C Web Accessibility Initiative (WAI) (YouTube) (links to the video in a broader playlist context)](https://www.youtube.com/watch?v=w-2nFEUBELo&list=PLhDEeYUfW02Qo4r2KlzagxZxhYcZADee-&index=5)
4. [The "Intent" section of "Understanding Success Criterion 1.3.1: Info and Relationships | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html#intent)

#### Using your browser's developer tools:

1. Inspect the element in question using your browser's developer tools.
2. Notice that neither of the following conditions is true:
    a. The element is an `<h*>` element (like `<h1>`,`<h2>`, `<h3>`, `<h4>`, `<h5>`, or `<h6>`)
    b. (Or) The element has `role="heading"` and the `aria-level` attribute set on it (with a value of "1", "2", "3", "4", "5", or "6").

#### Using a screen reader (such as VoiceOver, NVDA, JAWS, or TalkBack):

1. Activate the screen reader.
2. Navigate to the element (or pull up a list of headings on the page), and notice that the element is not announced (or included) as a heading.

### How To Fix The Issue

Update the element to be a heading element (`<h1>` - `<h6>`) (ensuring that you use a heading of the appropriate level). If it's not possible to change the element as a whole (due to technical or code access constraints), you can add `role="heading"` and the `aria-level` attribute (with a value of "1", "2", "3", "4", "5", or "6") to the element instead.

#### WCAG 2.2 Conformant Code Example

Not available/not applicable.
